mollie/api/objects/payment.py

```diff
@@ -55,9 +55,9 @@ def has_sequence_type_recurring(self):
 
     @property
     def checkout_url(self):
-        if '_links' not in self:
```

Comment:
De truc is natuurlijk dat je eigenlijk `_links`, `checkout` en `href` moet controleren voordat je het gebruikt. Misschien beter om gewoon te returnen en de KeyError af te vangen.